src/__tests__/e2e/correct-content.spec.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import { test, expect } from '@playwright/test'
+
+/**
+ * List of dev-portal paths and expected H1's.
+ */
+const table = [
+	['/hcp/tutorials/networking', 'HashiCorp Virtual Network'],
+	['/vault/tutorials/operations/configure-vault', 'Configure Vault'],
+	['/consul/tutorials/get-started-hcp/hcp-gs-deploy', 'Deploy HCP Consul'],
+]
```

Comment:
TODO: add more path+h1 pairs here.

I think having a hard-coded list here should be reasonable for the near future.

Depending on test slowness, I think 100+ tutorials should be a reasonably wide net that'll give us confidence during the next Next.js upgrade.

TODO: also add docs routes.

Reply:
💭 Definitely fine for now while we troubleshoot but if we want this test to always run in the future, it should be driven off real api data in case headings / filenames for tutorials chance.